gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/factories/GafferFactory.java

```diff
@@ -101,6 +104,32 @@ private static GafferSpec overrideGafferSpecConfig(final GafferSpec config, fina
         return opAuthoriser;
     }
 
+    private static ArrayList<Object> createOperationDeclaration(final GafferSpec config) {
+        ArrayList<Object> operations = new ArrayList<>();
+        if (config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY) != null) {
+            operations.add(config.getNestedObject(GAFFER_OPERATION_DECLARATION_KEY));
```

Comment:
Should this not be `addAll` as `operationDeclarations` is a list?

Reply:
I don't think so because `operationDeclarations` returns an object which then contains a list.